# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../gpe-julia/compile.patch
# Copyright (C) 2007 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- gpe-julia-0.0.6/main.c.vanilla	2007-08-12 12:04:35.000000000 +0200
+++ gpe-julia-0.0.6/main.c	2007-08-12 12:04:45.000000000 +0200
@@ -19,7 +19,6 @@
 #include <gpe/init.h>
 #include <gpe/picturebutton.h>
 #include <gpe/pixmaps.h>
-#include <gpe/render.h>
 #include <gpe/errorbox.h>
 
 static struct gpe_icon my_icons[] = {
